packages/datadog-plugin-winston/test/index.spec.js

```diff
@@ -54,7 +55,7 @@ describe('Plugin', () => {
 
     spy = sinon.spy()
 
-    class Transport extends winston.Transport {}
+    Transport = class extends winston.Transport {}
```

Comment:
Why is this change necessary? Doesn't seem to be accessed anywhere else. 🤔

Reply:
The change was needed for another change that was eventually reverted, so it should no longer be needed.

Reply:
I reverted the change.